scripts/issue/930/frax_stake_badgerfraxbp.py

```diff
@@ -0,0 +1,42 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+
+# https://etherscan.io/address/0x5a92EF27f4baA7C766aee6d751f754EBdEBd9fae#code#L722
+MIN_LOCK_TIME = 594000
+
+# https://etherscan.io/tx/0x926d29b92b4b1b2deef70e54e41d4478106eb8062a59ea58c74f628892c05710#eventlog
+PID = 35
```

Comment:
could take this from `vault.convex.BADGER_FRAXBP` instead.

Reply:
hmm i see now it is wrapped in its own class `VaultTypes` and not in `Convex`, why?

think moving it to `Convex` completely would be handier. dirty quick fix would be to add a `self.VaultTypes = VaultTypes` to `Convex`'s `__init__`, but a just dict would make things simpler, no?

Reply:
pushed into the class itself 43a1a257525ee6663976203c8b279ac13d77fcfb so it is more accessible